### ISSUE TITLE
NTR: Increased oocyte death

### ISSUE THREAD

**Issue by nicolevasilevsky** [obophenotype/human-phenotype-ontology#4945]
**Preferred term label:**
Increased oocyte death

**Definition (free text, please give PubMed ID)**
An increase in apoptosis of oocytes, the female germ cell (egg cell), which can occur before or after fertilization.

**Parent term (use hpo.jax.org/app)**
HP_0011017 'Abnormal cellular physiology'
(Not sure if this is the best parent)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
https://www.omim.org/entry/618550

See related ticket: https://github.com/monarch-initiative/hpo-annotation-data/issues/387

**Comment by pnrobinson** [comment]
I do not think that we can say apoptosis, since this was not specifically investigated. I have used the definition:
An increase in death of oocytes, the female germ cell (egg cell), which can be observed clinically in the setting of in vitro fertilization.PMID:
    30918116

**Comment by pnrobinson** [comment]
Added term as HP:0032571
